go.mod

```diff
@@ -95,7 +95,7 @@ require (
 	github.com/magiconair/properties v1.8.1 // indirect
 	github.com/mattn/go-shellwords v1.0.5 // indirect
 	github.com/mesos/mesos-go v0.0.9 // indirect
-	github.com/mholt/caddy v1.0.0
+	github.com/mholt/caddy v1.0.1
```

Comment:
the go references to github.com/mholt/caddy should be changed to github.com/caddyserver/caddy, not rewritten via a go.mod replace directive

Reply:
if we want to change where we pull this dependency, this line should change to `github.com/caddyserver/caddy v1.0.1`